blog.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-A-Dog</title>
+  </head>
+  <body>
+    <header>
+      <img class="logoimg" src="adopt-a-dog-spring2019-master/images/adoptadog-logo.svg" alt="adopt a dog logo"/>
```

Comment:
We'll cover later on how to handle this image correctly. For now, go ahead and delete this, and just use a background color on the header. Or, if you want to work ahead, you could research background images ;)